MapboxDirections/MBRouteOptions.swift

```diff
@@ -284,6 +284,11 @@ open class RouteOptions: NSObject, NSSecureCoding {
     }
     
     /**
+     `Bool` whether the the `ManeuverType` `.exitRoundabout` is included.
```

Comment:
Typo: the the. Suggestion:

> A Boolean value indicating whether the route includes a `ManeuverType.exitRoundabout` step when traversing a rotary or roundabout.
> 
> If this option is set to `true`, a route that traverses a roundabout includes both a `ManeuverType.takeRoundabout` step and a `ManeuverType.exitRoundabout` step. Otherwise, it only includes a `ManeuverType.takeRoundabout` step. This option is set to `false` by default.

It’s worth mentioning the default, because Objective-C developers won’t see the default value below.

Reply:
> A Boolean value indicating whether the route includes a `ManeuverType.exitRoundabout` or `ManeuverType.exitRotary` step when traversing a roundabout or rotary, respectively.
> 
> If this option is set to `true`, a route that traverses a roundabout includes both a `ManeuverType.takeRoundabout` step and a `ManeuverType.exitRoundabout` step; likewise, a route that traverses a large, named roundabout includes both a `ManeuverType.takeRotary` step and a `ManeuverType.exitRotary` step. Otherwise, it only includes a `ManeuverType.takeRoundabout` or `ManeuverType.takeRotary` step. This option is set to `false` by default.